### PR TITLE
1234

### PR DESCRIPTION
**Pull request by ijsm**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -66,5 +66,17 @@ div.navbar-collapse > div.navbar-nav > a.nav-link:hover {
 .foot-link {
 	margin-right: 1.25rem;
 }
+#image{
+	height: 15rem;
+	width: auto;
+	margin-top: 80px;
```
Comment (kritixilithos) [inline style.css]: Use `rem` instead of `px`

File: style.css [hunk 2]
```diff
@@ -66,5 +66,17 @@ div.navbar-collapse > div.navbar-nav > a.nav-link:hover {
...
+}
+
+#box{
+	height: 500px;
```
Comment (kritixilithos) [inline style.css]: Same here and the lines beneath this

File: style.css [hunk 3]
```diff
@@ -66,5 +66,17 @@ div.navbar-collapse > div.navbar-nav > a.nav-link:hover {
...
+	width: 500px;
+	border: 3px solid;
+
+}
 
 /* END: navbar/footer stuff */
```
Comment (kritixilithos) [inline style.css]: Respect this comment, shift your `#image {}` and `#box {}` to after the ` /* END: navbar/footer stuff */` comment for organisation

**Review by kritixilithos — CHANGES_REQUESTED** [review]

File: style.css (hunk 3 above)
Comment (ijsm) [reply style.css]: -_-

**Review by kritixilithos — APPROVED** [review]

**Pull request merged**